.github/workflows/cf-logs-fetcher.yml

```diff
@@ -0,0 +1,23 @@
+name: CF Deployment Logs
+on:
+  issue_comment:                                     
+    types: [created, edited]
+jobs:
+  fetch_comment_log:
+    if: ${{ (github.event.issue.pull_request) && (github.event.comment.user.login == 'cloudflare-pages[bot]' ) }}
+    runs-on: [ubuntu-latest]
+    steps:
+    
+      - name: Fetch & Print The Deployment Logs
+        uses: agoric-labs/cf-logs-fetcher@v1
+        with: 
+          cf_account_id: "0c4635effffcd7f36d1b9f0425a4367a"
+          cf_project: "documentation"
```

Comment:
What is `cf_account_id`? Is it some sensitive data? Should we add it in secrets?

Reply:
account id is safe to keep - since it is secret protected.